### DISCUSSION TITLE
Git inline blame too transparent

### DISCUSSION THREAD

**Discussion by emekaorji** [gitkraken/vscode-gitlens#2089] [Q&A] [answered]
There's no setting to increase the opacity of the inline blame.

With the theme I am using on vscode, I really am straining my eyes to see it.

**Comment by eamodio** [comment, marked as answer]
You can set the `gitlens.trailingLineForegroundColor` theme color in the `workbench.colorCustomizations` setting, e.g.:

```json
  "workbench.colorCustomizations": {
    "gitlens.trailingLineForegroundColor":"#ff0000"
  },
```

That will set the inline back to red. The value is RGBA.

**Reply by gomzila66-lab** [reply]
"workbench.colorCustomizations": {
    "gitlens.trailingLineForegroundColor":"#ff0000"
  },

**Comment by groengpx** [comment]
For the curious, the default is `#99999959`, which might help if you want to switch to the new the built-in experimental `git.blame.editorDecoration` and want to keep the same colors as with gitlens:

```json
"workbench.colorCustomizations": {
	"git.blame.editorDecorationForeground": "#99999959",
},
```

More generally, this is my config now with the built-in blame, to almost replicate gitlens' behaviour:

```json
"git.blame.statusBarItem.enabled": true,
"git.blame.statusBarItem.template": "${authorName}, ${authorDateAgo}",
"git.blame.editorDecoration.template": "${authorDateAgo} · ${subject}",
"git.blame.editorDecoration.enabled": true,
"workbench.colorCustomizations": {
	"git.blame.editorDecorationForeground": "#99999959",
},
```

I can't make the the built-in version replicate 'You' as author, and since my full git author name is quite long, I just leave it out.

**Reply by gomzila66-lab** [reply]
"git.blame.statusBarItem.enabled": true,
"git.blame.statusBarItem.template": "${authorName}, ${authorDateAgo}",
"git.blame.editorDecoration.template": "${authorDateAgo} · ${subject}",
"git.blame.editorDecoration.enabled": true,
"workbench.colorCustomizations": {
	"git.blame.editorDecorationForeground": "#99999959",
},
